cli/azd/.gitignore

```diff
@@ -5,3 +5,5 @@ azd-record.exe
 build
 resource.syso
 versioninfo.json
+azd.sln
```

Comment:
This is something that was getting written because I had the C# Dev Pack installed (for when I was writing the C# code to test this) and it wanted to create a `azd.sln` with all the C# code under the project for reasons that I'm sure are important to it, but I don't care about.